src/frontend/komponenter/Fagsak/Vedtak/Vedtaksalert.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+import { Alert } from '@navikt/ds-react';
+
+import { Behandlingstype, BehandlingÅrsak, type IBehandling } from '../../../typer/behandling';
+
+interface Props {
+    åpenBehandling: IBehandling;
+}
+
+export const Vedtaksalert: React.FunctionComponent<Props> = ({ åpenBehandling }) => {
+    const erSmåbarnstilleggEndringFramITid =
+        åpenBehandling.årsak === BehandlingÅrsak.SMÅBARNSTILLEGG_ENDRING_FRAM_I_TID;
+    const erMigreringFraInfotrygd = åpenBehandling.type === Behandlingstype.MIGRERING_FRA_INFOTRYGD;
+
+    if (erMigreringFraInfotrygd) {
+        return (
+            <Alert variant="info">
+                Du er inne på en migreringsbehandling og det sendes ingen vedtaksbrev.
+            </Alert>
+        );
+    } else if (erSmåbarnstilleggEndringFramITid) {
+        return <Alert variant="info">Du er inne på en behandling uten vedtaksbrev.</Alert>;
+    } else {
```

Comment:
Burde egentlig satsendring også gitt denne teksten i stedet for at man er inne på en teknisk behandling?